file/file-alfresco/src/main/java/uk/gov/justice/services/file/alfresco/requester/AlfrescoFileRequester.java

```diff
@@ -11,18 +11,20 @@
 import uk.gov.justice.services.common.configuration.GlobalValue;
 import uk.gov.justice.services.file.alfresco.common.AlfrescoRestClient;
 import uk.gov.justice.services.file.api.FileOperationException;
-import uk.gov.justice.services.file.api.requester.FileRequester;
+import uk.gov.justice.services.file.api.requester.StreamingFileRequester;
 
+import java.io.InputStream;
 import java.util.Optional;
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
+import javax.ws.rs.NotFoundException;
 import javax.ws.rs.ProcessingException;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.Response.StatusType;
 
 @ApplicationScoped
-public class AlfrescoFileRequester implements FileRequester {
+public class AlfrescoFileRequester implements StreamingFileRequester {
```

Comment:
Maybe just create a new class AlfrescoStreamgFileRequester and keep the interfaces separate? 
@matt-rich what do you think?